lib/programmemory.cpp

```diff
@@ -360,7 +360,6 @@ static void fillProgramMemoryFromAssignments(ProgramMemory& pm, const Token* tok
                         ++indentlevel;
                         continue;
                     }
-                    tok2 = cond->astParent()->previous();
```

Comment:
Why is this changed? If the condition is false we should be skipping the entire block.

Reply:
Ah wait, I see, it's a dead store.   Thats a bug. I think we need to continue here and not break. I think we can fix this in a later PR.